src/App.js

```diff
@@ -1,13 +1,62 @@
 import React from 'react';
-
 import './App.scss';
+import { getAll, get5First, getRed } from './api/goods';
+import { GoodsList } from './GoodsList';
+
+class App extends React.Component {
+  state = {
+    goods: [],
+  }
+
+  getGoods = (showOption) => {
+    showOption()
+      .then((goods) => {
+        this.setState({
+          goods: [...goods],
+        });
+      });
+  }
+
+  showAll = () => {
+    this.getGoods(getAll);
+  }
 
-// import { getAll, get5First, getRed } from './api/goods';
-// or
-// import * as goodsAPI from './api/goods';
+  showFive = () => {
+    this.getGoods(get5First);
+  }
 
-const App = () => (
-  <h1>Dynamic list of Goods</h1>
-);
+  showRed = () => {
+    this.getGoods(getRed);
+  }
```

Comment:
Can you make 1 function out of these 4, that will accept a callback as a parameter?